stories/cli/run.fmf

```diff
@@ -43,6 +43,38 @@ story: 'As a user I want to execute tests easily'
         - tmt run --continue
         - tmt run --abort
 
+/upgrade:
+    story:
+        As a tester I want to verify that a configured application
+        or service still correctly works after the system upgrade.
+    description: |
+        In order to enable developing tests for upgrade testing we
```

Comment:
Maybe we can be more clear that we speak about the "system tests", not unit tests that are written for each actor as well, but do not test the whole upgrade story.

Reply:
I see. This should now be clarified in 7cc2580.

Reply:
Cool, thanks.